compiler/compiler.go

```diff
@@ -391,8 +402,13 @@ func (c *compileCtx) getType(name string) (protobuf.Type, error) {
 		}
 
 		for t := range container {
-			if t.Name() == name {
-				return t, nil
+			if name == "array" {
+				c.addImportForType("google.protobuf.ListValue")
+				return c.getArrayType(name), nil
+			} else {
```

Comment:
If this structure is needed, it'd be cleaner to do an `else if` or a `switch` statement here.